docs/dg/dev/third-party-module-development/ensure-quality.md

```diff
@@ -0,0 +1,95 @@
+| spryker-dev-console dev:validate-module-schemas YourCompanyName.YourModuleName            | Validates Propel schema definitions.                                    |
+| spryker-dev-console dev:validate-module-databuilders YourCompanyName.YourModuleName       | Validates databuilder XML files.                                        |
+| spryker-dev-console dev:module-meta-files -d -v YourCompanyName.YourModuleName            | Validates that all the mandatory meta files exist.                      |
+| spryker-dev-console dev:composer:update-json-files -d -v YourCompanyName.YourModuleName   | Validates that all the necessary module dependencies are added.         |
+| console dev:composer:validate-json-files YourCompanyName.YourModuleName                   | Validates composer.json file to make sure that it's structure is valid. |
+| console code:sniff:style -m YourCompanyName.YourModuleName                                | Sniffs and fixes code style.                                            |
+| spryker-dev-console dev:validate-module-transfers -m YourCompanyName.YourModuleName       | Validates DTO definition for the module.                                |
+| console code:sniff:architecture -v -m YourCompanyName.YourModuleName                      | Validates module architecture to find a common architecture mistakes.   |
+| console code:phpstan -m YourCompanyName.YourModuleName                                    | Runs PHPStan static analyzer to ensure the code quality.                |
+
+## GitHub CI file example 
+
+Make sure that your module CI runs all the command above
+
+```yaml
+name: CI
+
+on:
+    push:
+        branches:
+            - 'master'
+    pull_request:
+    workflow_dispatch:
+
+jobs:
+    validation:
+        name: Validation
+        runs-on: ubuntu-latest
+
+        steps:
+            - uses: actions/checkout@v3
+
+            - name: Setup PHP
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: '8.2'
+                  extensions: mbstring, intl, bcmath
+                  coverage: none
+
+            - name: Composer Install
+              run: composer install --prefer-dist --no-interaction --profile
+
+            - name: Run validation
+              run: composer validate
+
+            # RUN ALL THE COMMANDS FROM THE TABLE ABOVE
+    lowest:
+        name: Prefer Lowest
+        runs-on: ubuntu-latest
+
+        steps:
+            - uses: actions/checkout@v3
+
+            - name: Setup PHP
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: '8.1'
+                  extensions: mbstring, intl, bcmath
+                  coverage: none
+
+            - name: Composer Install
+              run: composer install --prefer-dist --no-interaction --profile
+
+            - name: Composer Update
```

Comment:
no validation?

Reply:
It has been taken from the standalone module validation template https://github.com/spryker/product/blob/master/.github/workflows/ci.yml